src/main/kotlin/org/rust/ide/formatter/RustLanguageCodeStyleSettingsProvider.kt

```diff
@@ -27,7 +27,10 @@ class RustLanguageCodeStyleSettingsProvider : LanguageCodeStyleSettingsProvider(
     override fun customizeSettings(consumer: CodeStyleSettingsCustomizable, settingsType: SettingsType) {
         when (settingsType) {
             BLANK_LINES_SETTINGS         -> {
-                // TODO Implement this.
+                consumer.showStandardOptions(
+                    "KEEP_LINE_BREAKS",
+                    "KEEP_BLANK_LINES_IN_DECLARATIONS",
+                    "KEEP_BLANK_LINES_IN_CODE")
```

Comment:
Yep it just do not display given option in code style settings panel; so it's pretty easy to verify this anyway.